index.js

```diff
@@ -36,12 +40,20 @@ const wrapWord = (rows, word, columns) => {
 
 		if (ESCAPES.has(character)) {
 			isInsideEscape = true;
-		} else if (isInsideEscape && character === 'm') {
-			isInsideEscape = false;
-			continue;
+			isInsideLinkEscape = word.slice(index).startsWith(`${character}${ANSI_ESCAPE_LINK}`);
 		}
 
 		if (isInsideEscape) {
+			/* istanbul ignore if: cannot enter on terminals not supporting hyperlinks */
+			if (isInsideLinkEscape) {
+				if (character === ANSI_ESCAPE_BELL) {
+					isInsideEscape = false;
+					isInsideLinkEscape = false;
+				}
+			} else if (character === 'm') {
```

Comment:
`'m'` should be defined as constant.

Reply:
Not really part of my change but sure I'll fix it.